packages/cloud2edge/Chart.yaml

```diff
@@ -11,8 +11,8 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 apiVersion: v1
-version: 0.2.4
-appVersion: 0.2.4
+version: 0.3.0
+appVersion: 0.3.0
```

Comment:
@kalinkostashki I would suggest to only increase to `0.2.5` here - as this can be seen as "bugfix" of the chart.
When using new Ditto/Hono major versions we can update to `0.3.0`